app/src/main/java/de/rochefort/childmonitor/audio/G711UCodec.kt

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2009 The Android Open Source Project
```

Comment:
License comment needs to be adjusted

Reply:
In which way?

Reply:
This was the comment from the file I copied originally, which was a java file. Since it is now changed quite drastically, I am assuming that we needed to somehow state that the current code is based on a state that is subject to the above license statement, but was now converted to kotlin.
Probably best below my "Taken from" notice.

Reply:
I've added a short notice.
I'd say, that without different notice, the code and the changes are subject to the Apache 2.0 license.

Reply:
I always struggle with licensing questions. In this case I think that neither the kotlin conversion nor the method removal justifies any copyright claims on the resulting code. I would hence not add my own license statement and leave it licensed under the apache license.

I am still unclear about the exact wording I'd like to have in the additional note though. 

How about this:
```
 *
 * Originally taken from https://android.googlesource.com/platform/external/nist-sip/+/6f95fdeab4481188b6260041b41d1db12b101266/src/com/android/sip/media/G711UCodec.java
 * Modifications to the original code:
 *   - Source conversion to kotlin
 *   - Removal of unused method getSampleCount
```

Reply:
Well, it's your project :). I wouldn't make any claims, but the license would allow for that.

My understanding is, it is sufficient to say that it has been modified:
> You must cause any modified files to carry prominent notices stating that You changed the files;

So no need to keep a change-log. It is only to avoid confusion.

I.e.:
```
 * Originally taken from https://android.googlesource.com/platform/external/nist-sip/+/6f95fdeab4481188b6260041b41d1db12b101266/src/com/android/sip/media/G711UCodec.java
 * Adopted to the needs of the Child Monitor project.
```

The example from an Apache NOTICE file as [reference](https://infra.apache.org/licensing-howto.html#example-notice).

But it is your call, of course.

Reply:
I like your proposal, let's go with this!